python/openai/tests/test_chat_completions.py

```diff
@@ -564,9 +564,15 @@ def test_chat_completions_invalid_chat_tokenizer(
 
         # Pick a tokenizer with no chat template defined
         invalid_chat_tokenizer = "gpt2"
-        app = setup_fastapi_app(
-            tokenizer=invalid_chat_tokenizer, server=server, backend=backend
-        )
+        try:
+            app = setup_fastapi_app(
+                tokenizer=invalid_chat_tokenizer, server=server, backend=backend
+            )
+        except OSError as e:
+            expected_msg = f"We couldn't connect to 'https://huggingface.co' to load this file, couldn't find it in the cached files and it looks like {invalid_chat_tokenizer} is not the path to a directory containing a file named config.json."
```

Comment:
This is fine for now, but just be wary if the message changes slightly in the future it might not be an exact match.

Reply:
Yes. I want to be cautious on this first, to avoid skipping the test for anything else.

If the message varies in the future, I think we can update the condition to detect if the network communication is disabled, i.e. `HF_HUB_OFFLINE`.